### DISCUSSION TITLE
empty window with ctkmessagebox

### DISCUSSION THREAD

**Discussion by aeris07** [Akascape/CTkMessagebox#47] [Q&A] [answered]
Hello, there
When I try to use ctkmessagebox in a function and call it, it displays an empty window along with the messagebox,

Example code:
from CTkMessagebox import CTkMessagebox
def show_info():
    CTkMessagebox(title="Info", message="This is a CTkMessagebox!")
show_info()

![resim](https://github.com/Akascape/CTkMessagebox/assets/129675592/192defe4-b4a8-48e2-a0a5-09de04224020)

I was expecting  to display only ctkmessagebox

**Comment by Akascape** [comment, marked as answer]
You should have a root window in order to display any messagebox.

**Comment by aeris07** [comment]
I got it, 
Thank you for the answer and the widget by the way.
